### PR TITLE
Fix typos discovered by codespell

### PR DESCRIPTION
**Pull request by cclauss**
> codespell --skip="*.json,*.rtf" \
    --ignore-words-list=crate,delet,inout,iterm,jumo,lightening,othe,ser,tabe,te,tese,thi,updat
* https://github.com/codespell-project/codespell/blob/master/README.rst

Release Notes:

- N/A

### CODE REVIEW

**Comment by cclauss** [comment]
@cla-bot check

File: crates/collab/migrations/20231009181554_add_release_channel_to_rooms.sql [hunk 1]
```diff
@@ -1 +1 @@
-ALTER TABLE rooms ADD COLUMN enviroment TEXT;
+ALTER TABLE rooms ADD COLUMN environment TEXT;
```
Comment (cclauss) [inline crates/collab/migrations/20231009181554_add_release_channel_to_rooms.sql:1]: Caution: This changes SQL code.  Is this correct?

File: crates/collab/migrations/20231009181554_add_release_channel_to_rooms.sql [hunk 2]
```diff
@@ -1 +1 @@
-ALTER TABLE rooms ADD COLUMN enviroment TEXT;
```
Comment (mikayla-maki) [inline crates/collab/migrations/20231009181554_add_release_channel_to_rooms.sql:1]: This one has to stay

File: crates/collab/migrations/20231009181554_add_release_channel_to_rooms.sql (hunk 1 above)
Comment (mikayla-maki) [reply crates/collab/migrations/20231009181554_add_release_channel_to_rooms.sql:1]: Since this is in the migrations, we have to leave it 🙃

**Comment by nathansobo** [comment]
Thnaks!

**Pull request merged**